src/__tests__/permissionSelectors-test.js

```diff
@@ -0,0 +1,100 @@
+/* @flow strict-local */
+import invariant from 'invariant';
+
+import { tryGetActiveAccountState } from '../selectors';
+import {
+  Role,
+  type RoleT,
+  CreateWebPublicStreamPolicy,
+  type CreateWebPublicStreamPolicyT,
+} from '../api/permissionsTypes';
+import { getCanCreateWebPublicStreams } from '../permissionSelectors';
+import rootReducer from '../boot/reducers';
+import { EVENT_USER_UPDATE } from '../actionConstants';
+import * as eg from './lib/exampleData';
+
+describe('getCanCreateWebPublicStreams', () => {
+  const { AdminOrAbove, ModeratorOrAbove, Nobody, OwnerOnly } = CreateWebPublicStreamPolicy;
+  const { Owner, Admin, Moderator, Member, Guest } = Role;
+
+  test('returns false when webPublicStreamsEnabled is false', () => {
+    const globalState = eg.reduxStatePlus({
+      realm: { ...eg.plusReduxState.realm, webPublicStreamsEnabled: false },
+    });
+
+    const state = tryGetActiveAccountState(globalState);
+    invariant(state, 'expected per-account state');
+
+    expect(getCanCreateWebPublicStreams(state)).toBeFalse();
```

Comment:
This should make sure the other conditions are in place so that it *would* be allowed if not for this setting.

From a goals perspective: the purpose of this test is to exercise the `webPublicStreamsEnabled` condition in the function. If I comment that condition right out:
```diff
-  if (!webPublicStreamsEnabled || !enableSpectatorAccess) {
-    return false;
-  }
+  // if (!webPublicStreamsEnabled || !enableSpectatorAccess) {
+  //   return false;
+  // }
```
then the test still passes. So the test isn't accomplishing its purpose.

Concretely, fixing this means something like make the self-user an owner and the policy `ModeratorOrAbove`.

(If the test did successfully fail on knocking that code out, then it'd be because of values we happen to have in the default example data, and we'd still want to explicitly set them here.)